### ISSUE TITLE
Option unwrap on none value

### ISSUE THREAD

**Issue by prildy** [sendyhalim/lezeh#15]
Hi I got this error when doing **merge-feature-branches**

```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /Users/runner/work/lezeh/lezeh/lezeh-deployment/src/lib/cli.rs:167:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

no clue on which repo or which pull request but some pull requests got merged anyway.

**Comment by sendyhalim** [comment]
Hi @prildy ,

looking at the error log, are u building/running lezeh directly from the source code? at what commit hash did u run lezeh? The error is because I mistakenly force unwrapping optional, if it's confirmed (the same issue that I pointed) then it should be fixed on master and 0.8.2

https://github.com/sendyhalim/lezeh/blob/master/lezeh-deployment/src/lib/cli.rs#L167

**Comment by sendyhalim** [comment]
Just discussed in private that the bug is from 0.8.0 which should be fixed by 0.8.2, I'm closing this issue for now. Feel free to reopen.
